src/engine.cpp

```diff
@@ -611,8 +611,8 @@ void CalcEngineReliability(Engine *e, bool new_month)
 		re = Engine::Get(re->info.variant_id);
 	}
 
-	uint age = re->age;
-	if (new_month && re->index > e->index && age != MAX_DAY) age++; /* parent variant's age has not yet updated. */
+	uint32 age = re->age;
```

Comment:
re->age is of type `int32`, why make it `uint32` here?

Reply:
We compare this local variable against a variety of duration phases, which are all unsigned. This PR came about when trying to fix signed/unsigned comparison annotations in #10746, and I wanted to avoid that headache again.